subsys/bluetooth/common/rpa.c

```diff
@@ -23,34 +23,33 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_rpa);
 
-#if defined(CONFIG_BT_CTLR) && defined(CONFIG_BT_HOST_CRYPTO)
+#if defined(CONFIG_BT_CTLR_CRYPTO) && defined(CONFIG_BT_HOST_CRYPTO)
 #include "../controller/util/util.h"
 #include "../controller/hal/ecb.h"
-#endif /* defined(CONFIG_BT_CTLR) && defined(CONFIG_BT_HOST_CRYPTO) */
+#endif /* CONFIG_BT_CTLR_CRYPTO && CONFIG_BT_HOST_CRYPTO */
```

Comment:
Why do we need to enable both host and controller crypto? Should this have been a ` || `?

Reply:
Ditto for the below checks